src/redist/targets/GenerateMSIs.targets

```diff
@@ -344,9 +344,11 @@
     </ItemGroup>
     <PropertyGroup>
       <LatestTemplateMsiInstallerFile>@(LatestTemplateInstallerComponent->'%(MSIInstallerFile)')</LatestTemplateMsiInstallerFile>
+      <UpgradePoliciesSrcPath>$(PkgMicrosoft_DotNet_Build_Tasks_Installers)\build\wix\bundle\upgradePolicies.wxs</UpgradePoliciesSrcPath>
```

Comment:
Is `PkgMicrosoft_DotNet_Build_Tasks_Installers` guaranteed to not have any invalid PATH characters as defined by https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getinvalidpathchars?view=net-8.0#system-io-path-getinvalidpathchars?

Reply:
yup, NuGet generates path properties for you when you restore packages and generates well known msbuild properties based on the pacakge ID. We simply use this to find the matching arcade package and locate the source file that comes from the Arcade SDK - related to this PR: https://github.com/dotnet/arcade/pull/15047